src/psyclone/domain/lfric/arg_ordering.py

```diff
@@ -455,6 +479,22 @@ def stencil(self, arg, var_accesses=None):
 
         '''
 
+    @abc.abstractmethod
+    def stencil_2d(self, arg, var_accesses=None):
+        '''Add general stencil information associated with the argument 'arg'
```

Comment:
This docstring is currently identical to that for `stencil()`. Presumably it shouldn't be?

Reply:
Changed to reflect 2D stencil info